src/components/Layout/Navbar.js

```diff
@@ -0,0 +1,15 @@
+import StyledNav from './stylednav.js';
+import StyledNavLink from './stylednavlink.js';
+
+export default function Navbar() {
+	return (
+		<StyledNav>
+			<StyledNavLink to="/">
+				<i className="fa-solid fa-house"></i>
+			</StyledNavLink>
+			<StyledNavLink to="/planner">
+				<i className="fa-solid fa-calendar-days"></i>
+			</StyledNavLink>
```

Comment:
Maybe add some aria-labels here for better accessibility:
```suggestion
			<StyledNavLink to="/" aria-label="Home">
				<i className="fa-solid fa-house"></i>
			</StyledNavLink>
			<StyledNavLink to="/planner" aria-label="Planner">
				<i className="fa-solid fa-calendar-days"></i>
			</StyledNavLink>
```

Reply:
Hi Lene,
thanks for your review!
I forgot the accessibility point regarding the navbar, I implemented this and the other things now.